src/pages/styles.module.css

```diff
@@ -107,3 +102,33 @@
 .floatingNews li {
   margin-top: 10px;
 }
+
+/* carousel */
+
+.galleryImage{
+  /* width: var(--ifm-container-width); */
+  /* height: calc( var(--ifm-container-width) * 0.5 ); */
+  max-width: 100%;
+  max-height: 100%;
+  display: block;
+  margin: auto;
+  object-fit: contain;
+}
+
+.carouselContent {
+  width: var(--ifm-container-width);
+  height: calc( var(--ifm-container-width) * 0.5 );
+  color: #fff;
+  line-height: 160px;
+  text-align: center;
+};
+
+@media screen and (max-width: 998px) {
+  /* .galleryImage{
+    max-width: 100vw;
+  } */
+  .carouselContent {
```

Comment:
It seems there is a large gap between an image and the buttons when browsed on mobile devices. How about reducing the height as well on smaller screens?

![Screen Shot 2022-04-13 at 09 23 08](https://user-images.githubusercontent.com/9922882/163194088-0f3f98bd-a68d-41ae-bcf4-0f6d83f7141a.png)